vcpkg/github-integration.md

```diff
@@ -0,0 +1,94 @@
+---
+title: GitHub Integration
+description: Information on the available vcpkg integration points with GitHub
+author: michaelbprice
+ms.author: miprice
+ms.date: 06/15/2023
+---
+
+# Available GitHub Integrations
+
+## Caching vcpkg-built binaries for your GitHub Actions workflows
+
+Repositories that use GitHub Actions for continuous integration can take advantage of [vcpkg's binary caching feature](./users/binarycaching.md) to reduce the amount of time it takes to build their projects. There are two binary cache providers that are available to GitHub repositories.
+
+### The GitHub Actions cache provider
+
+[!INCLUDE [experimental](../includes/experimental.md)]
+
+The [GitHub Actions cache](https://docs.github.com/actions/using-workflows/caching-dependencies-to-speed-up-workflows) is intended to store a repository's intermediate build files that don't change often between jobs or workflow runs. GitHub provides a few different [tools to manage your Actions caches](https://docs.github.com/actions/using-workflows/caching-dependencies-to-speed-up-workflows#managing-caches), which include REST APIs and an extension to the `gh` command line tool, so that you can optimize caches for your workflow. vcpkg's integration with GitHub Actions cache is through the [`x-gha` binary source provider](./users/binarycaching.md#gha).
```

Comment:
Mentioned previously - don't rely on inline links, i.e. don't rely on link text as part of the sentence.  Say what you want to say about something, and then provide a link for more information. Imagine removing all the links - the article should still make sense. Part of the problem with relying on inline links is that it makes the doc hard to read because sometimes these links are used as a crutch for explaining the author should have done.  Then, you could provide links to provide additional context, if necessary. Another reason to avoid inline links is that it makes the article hard to comprehend. The reader has to (or thinks they have to) ricochet from link to link to assemble the full picture. The article author should do that for them and just provide links for more depth.

Reply:
I've done a scrub through the document and replaced in-sentence links with follow-up sentences that still try to provide useful links, while trying to avoid the "click here" accessibility mistake. It makes things a little bit repetitive, especially for short paragraphs, but hopefully this is better.

I did leave one in place (link to instructions on how to enable the dependency graph for private repos) because it already felt like this sort of "Go here for more" phrasing already.